dev-util/imediff/imediff-3.3.1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_SINGLE_IMPL=1
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..13} )
+PYTHON_REQ_USE="ncurses"
+
+inherit distutils-r1
+
+DESCRIPTION="2-way/3-way merge tool (CLI, Ncurses)"
+HOMEPAGE="https://github.com/osamuaoki/imediff"
+SRC_URI="https://github.com/osamuaoki/${PN}/archive/upstream/${PV}.tar.gz -> ${P}.tar.gz"
+
+S="${WORKDIR}/${PN}-upstream-${PV}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
```

Comment:
Being a new major version, it's best to reset the keywords, so I'll do that now. I've tested on amd64, so I'll keep that.

Reply:
Ok. Thanks !